platform/linux-generic/m4/odp_crypto.m4

```diff
@@ -31,10 +31,24 @@ AS_IF([test "x$with_crypto" == "xarmv8crypto"],
       [PKG_CHECK_MODULES([AARCH64CRYPTO], [libAArch64crypto])
        AARCH64CRYPTO_PKG=", libAArch64crypto"
        AC_SUBST([AARCH64CRYPTO_PKG])])
+
 AC_CONFIG_COMMANDS_PRE([dnl
 AM_CONDITIONAL([WITH_ARMV8_CRYPTO], [test "x$with_crypto" == "xarmv8crypto"])
 ])
 
+##########################################################################
+# Multi-buffer IPSec library implementation
+##########################################################################
+ipsecmb_support=0
+AS_IF([test "x$with_crypto" == "xipsecmb"],
+      [AC_CHECK_HEADERS([ipsec-mb.h],
+                        [ipsecmb_support=1],
+                        [AC_MSG_ERROR([IPSec MB library not supported on this platform])])])
+
+AC_CONFIG_COMMANDS_PRE([dnl
+AM_CONDITIONAL([WITH_IPSECMB_CRYPTO], [test "x$with_crypto" == "xipsecmb"])
+])
```

Comment:
Multi-buffer crypto library dependency is never added to `LIBS`. You can use `m4/odp_openssl.m4` as an example how to do this. New dependency should also be added to `platform/linux-generic/libodp-linux.pc.in`.

Reply:
Updated.